samples/bluetooth/beacon/src/main.c

```diff
@@ -85,4 +85,5 @@ void main(void)
 	if (err) {
 		printk("Bluetooth init failed (err %d)\n", err);
 	}
+	return 0;
```

Comment:
One comment that may only apply for BT, but for the BT subsystem we usually keep an empty line after an ending `}`, so this would be

```suggestion
	}
	
	return 0;
```

Reply:
These changes were done using coccinelle; I see a bunch of value in not messing with the results of that where possible as the tool is less likely to generate mistakes than manual editing? Perhaps a manual clean up pass in the series which adjusts formatting could be done though?

Reply:
> These changes were done using coccinelle; 

It does not have an option to specify empty line after ending bracets?